build/ci/pr/Jenkinsfile

```diff
@@ -126,6 +123,24 @@ REPOSITORY = "$GCLOUD_PROJECT"
 TESTS_MATCH = TestSmoke
 SKIP_DOCKER_COMMAND = false
 IMG_SUFFIX = -ci
+E2E_JSON = true
+EOF
+    """
+}
+
+def createDeployerConfig() {
+    sh """
+        cat >deployer-config.yml <<EOF
+id: gke-ci
+overrides:
+  kubernetesVersion: "1.12"
```

Comment:
I forget to update the deployer config of this job definition in #1895.
We can remove `kubernetesVersion: "1.12"` to use the default version defined in `hack/deployer/config/plans.yml`.
```suggestion
```

Reply:
Done